js_src/package.json

```
@@ -4,38 +4,24 @@
   "version": "1.0.0-alpha.0",
   "private": true,
   "license": "UNLICENSED",
-  "main": "src/_react-testing-library.js",
+  "main": "src/index.js",
   "files": [
     "src"
   ],
-  "module": "src/_react-testing-library.js",
+  "module": "src/index.js",
   "prodBundle": "../lib/js/react-testing-library.js",
   "scripts": {
-    "build": "yarn build:bundle",
-    "build:bundle": "rollup -c"
+    "build": "vite build",
+    "build:bundle": "vite build"
   },
   "dependencies": {
-    "core-js": "3",
-    "@testing-library/dom": "^7.30.0",
-    "@testing-library/react": "^11.2.5",
+    "@testing-library/dom": "^8.0.0",
+    "@testing-library/react": "^13.0.0",
```

#nit could we up these bounds to match what we resolved to, to help ensure we don't go backwards?
```suggestion
    "@testing-library/dom": "^8.20.1",
    "@testing-library/react": "^13.4.0",
```